app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -157,6 +160,11 @@ description: |
   elsif nic_delay
     options.push("nicdelay=#{nic_delay} linksleep=#{nic_delay}")
   end
+
+  # add user specified custom options
+  if custom_options
```

Comment:
This isn't even needed anymore, since `array.concat([])` is equal to `array`. Also, `if []` evaluates to `true` (unlike Python), so it's effectively no difference anyway.

Reply:
Done :)